MANDELBR.BAS

```diff
@@ -43,14 +45,46 @@ FOR py = 0 TO h - 1
         NEXT

```

Comment:
A colleague pointed out that on line 38 we do the x*x, y*y twice per iteration.
We could save an extra ^2 calculation per pixel  if we calculate that only once and store it in an x2 / y2 variable.

Reply:
That's true. I have other projects I'm working on right now.

Maybe you can do this one?

Reply:
Yep. See if I can get some time tonight.

Reply:
Yep, got it done.
Thanks for your help! Was fun!